common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -228,7 +230,10 @@ div.problem {
 
     input[type="radio"],
     input[type="checkbox"] {
+      @include left($baseline/4);
       @include margin(($baseline/4) ($baseline/2) ($baseline/4) ($baseline/4));
+      position: absolute;
+      top: $baseline*.6;
```

Comment:
I tested this. Looks like the radio button is a bit too low. This could just be top: $baseline;